packages/app/src/app/components/TeamAvatar/TeamAvatar.tsx

```diff
@@ -15,29 +15,51 @@ export const backgrounds = [
 
 interface TeamAvatarProps {
   name: string;
-  size?: 'small' | 'big';
+  size?: 'small' | 'big' | 'bigger';
   className?: string;
+  avatar?: string;
+  style?: any;
 }
 
 export const TeamAvatar = ({
   name,
   size = 'big',
   className,
+  avatar,
+  ...props
```

Comment:
Is there an explicit need for the use of `...props` here? I would prefer to not use it too much because it adds a bit of ambiguity (you don't know what can be passed to this component anymore) and causes a small performance hit, because it's transpiled to this code: https://babeljs.io/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAhjAvDAFAbwEYBoYDp8AOATiARAL4CUSAfKgFAwwA8AJgJYBuMGim5MNPlzFSFGAHoa9SkA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Creact%2Cstage-2%2Cenv&prettier=false&targets=&version=7.10.4&externalPlugins=.

If we can explicitly name the props I would always go for that approach, but yeah, only if possible.

Reply:
I do that mostly because of style props, so we can always style everything :/

Changed to passing style